Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -93,3 +93,72 @@ public extension Message {
   }
 }
 
+
+public extension Array where Iterator.Element: Message {
```

Comment:
Hm. Good point. I understand the concern, and share it.

On the one hand, it is somewhat sandboxed by being a constrained extension. If there were a legitimate ambiguity scenario, it's not impossible to shift things out to a file that doesn't import `SwiftProtobuf`.

One the other hand, a static funcs definitely seems not terrible, if that seems within the API flavor of the library.